smartsim/_core/control/manifest.py

```diff
@@ -127,24 +113,16 @@ def _check_names(deployables: t.List[t.Any]) -> None:
     @staticmethod
     def _check_types(deployables: t.List[t.Any]) -> None:
         for deployable in deployables:
-            if not isinstance(deployable, (SmartSimEntity, EntitySequence)):
+            if not isinstance(deployable, SmartSimEntity):
                 raise TypeError(
-                    f"Entity has type {type(deployable)}, not "
-                    + "SmartSimEntity or EntitySequence"
+                    f"Entity has type {type(deployable)}, not " + "SmartSimEntity"
```

Comment:
Style nit, while you're here, lol
```suggestion
                    f"Entity has type {type(deployable)}, not SmartSimEntity"
```

Reply:
Will be caught in [this ticket](https://github.com/CrayLabs/Poseidon/issues/830)